app/assets/stylesheets/teams.css

```diff
@@ -4,11 +4,192 @@
 */
 
 label.required:after {
-  content: "*";
-  vertical-align: 0.5em;
-  font-size: 0.75em;
-  font-weight: normal;
+    content: "*";
+    vertical-align: 0.5em;
+    font-size: 0.75em;
+    font-weight: normal;
 }
-select.filterSelect{
-  display: block;
+
+/* Overwrite container from application.html.erb for background image only*/
+body > .background-layer > .container {
+    width: 100%;
+    max-width: 100% !important;
+    padding: 0 0;
+}
+
+#teamProfile {
+    background-size: cover;
+    background-position: bottom center;
+    width: 100%;
+    background-image: url(https://owncloud.hpi.de/index.php/s/fERWNGqZbm6ud1p/download);
+    background-repeat: no-repeat;
+    min-height: 50vh;
+    padding-bottom: 300px;
+}
+
+#teamProfileImage {
+    width: 200px;
+    height: 200px;
+}
+
+#teamDescription {
+    overflow: hidden;
+    word-wrap: break-word;
+}
+
+h1, h2 {
+    color: #1CA9E5;
+    font-weight: 500;
+}
+
+h1 {
+    margin-top: 0;
+}
+
+.genericButton, .genericButton:hover, .genericButton:focus {
+    outline: none !important;
```

Comment:
To my mind, removing the `:focus`-outline property **without adding another visual indication for focused elements** is quite an anti-pattern since it reduces the accessibility of the website significantly. See http://www.outlinenone.com for further explanation. 

Currently, the “Select all”, “Select none”, “Edit” and “Delete” buttons have a `:focus` state that looks exactly the same as the non-`:focus` state. You can inspect this e. g. by Tab-navigating (using the Tab key on you keyboard) over the Team page or by using the Chrome Developer Tools' “Force element state” panel in the Elements tab:

<img width="1440" alt="ohne titel" src="https://user-images.githubusercontent.com/807665/35776736-68e3ea42-09a2-11e8-835a-5c7800a71d02.png">

@deeps96 Please make the `:focus` states of the buttons used on the Team page distinguishable from the normal inactive state. A simple fast solution would be to extend the existing `:hover` property changes you introduced to changing not only the `:hover` state, but also the `:focus` states (e. g. by lightening up the buttons' background colors).